build-image/first-boot.bash

```diff
@@ -212,12 +212,13 @@ echo -n "$(timestamp) [openHABian] Waiting for openHAB to become ready on ${HOST
 # this took ~130 seconds on a RPi2
 if ! tryUntil "curl --silent --head http://${HOSTNAME:-openhab}:8080/start/index |& grep -qs 'HTTP/1.1 200 OK'" 20 10; then echo "OK"; else echo "FAILED"; exit 1; fi
 
-echo "$(timestamp) [openHABian] Visit the openHAB dashboard now: http://${HOSTNAME:-openhab}:8080"
-echo "$(timestamp) [openHABian] To gain access to a console, simply reconnect."
-echo "$(timestamp) [openHABian] First time setup successfully finished."
+echo "$(timestamp) [openHABian] First time setup successfully finished. Rebooting your system!"
+echo "$(timestamp) [openHABian] After rebooting the openHAB dashboard will be available at: http://${HOSTNAME:-openhab}:8080"
+echo "$(timestamp) [openHABian] After rebooting to gain access to a console, simply reconnect using ssh."
 sleep 12
```

Comment:
remove the sleeps ?

Reply:
Why are they there?

Reply:
no clue, I guess they originate from the guy mues-lee that wrote the webif stuff.

Reply:
K. I'll remove them, I see no reason to keep them.

Reply:
Actually the first one has a purpose, to make sure the last couple of messages actually show in the log on the website. Therefore, I will leave it but remove the other one.

Reply:
they will either way as "reboot" stops OH first, ZRAM next so enough time

Reply:
No they won't because what is happening is he is delaying it so the final messages appear in the Web log then turning off the server. The first sleep is needed for UX the second one can go though.